tests/integration/test_xrdsst.py

```diff
@@ -1144,6 +1153,7 @@ def test_run_configuration(self):
 
         LocalGroupTest(self).test_run_configuration()
         RenewCertificate(self).test_run_configuration()
+        self.step_make_owner()
```

Comment:
It would be less confusing if the step name would also include "client", e.g step_client_make_owner